apps/archive/archive_media.py

```diff
@@ -45,12 +45,10 @@ def on_create(self, docs):
             doc['media'].seek(0)
             file_type = content_type.split('/')[0]
             if file_type == 'video' and app.config.get("VIDEO_SERVER_ENABLE"):
-                if not self.videoEditor.check_video_server():
-                    raise SuperdeskApiError(message="Cannot connect to videoserver", status_code=500)
                 # upload media to video server
                 res, renditions, metadata = self.upload_file_to_video_server(doc)
                 # get thumbnails for timeline bar
-                self.videoEditor.get_timeline_thumbnails(doc.get('media'), 40)
+                self.video_editor.create_timeline_thumbnails(doc.get('media'), 60)
```

Comment:
I wonder if that 60 should be rather a constant or a config option

Reply:
I will make it constant for now